### PR TITLE
explain API options

### LINKED ISSUE #155: Add explanation of API options - ETH, CLIQUE, etc
User feedback: 
The documentation does not specifically say what the commands for ETH,ADMIN,API,CLIQUE etc do for RPC, this is more of a nice thing as one can figure it out but its not explicitly stated I believe

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [x] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [x] you fixed all the issues raised by the tests, if any.
- [x] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change

Explain each of the API namespace options (ADMIN, NET, etc).

## Issue fixed

fixes #155 

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

## Testing

see https://hyperledger-besu--655.org.readthedocs.build/en/655/

### CODE REVIEW

File: docs/Reference/API-Methods.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ description: Hyperledger Besu JSON-RPC API methods reference
 
 ## `ADMIN` methods
 
+The `ADMIN` API methods allow you to control your Besu instance.
```
Comment (bgravenorst) [inline docs/Reference/API-Methods.md:20]: Maybe, "provides administrative functionality to manage your node."?

File: docs/Reference/API-Methods.md [hunk 2]
```diff
@@ -4949,6 +4964,8 @@ None
 
 ## `IBFT` 2.0 methods
 
+The `IBFT` API methods provide access to the IBFT 2.0 consensus engine.
```
Comment (bgravenorst) [inline docs/Reference/API-Methods.md:4967]: Perhaps link to any IBFT 2.0 concept info?

**Review by bgravenorst — APPROVED** [review]
Minor suggestions.

**Review by atoulme — APPROVED** [review]
LGTM

**Pull request merged**